blark/transform.py

```diff
@@ -2604,6 +2604,23 @@ def __str__(self):
         return f"{invoc};"
 
 
+@dataclass
+@_rule_handler("function_call_statement_list")
```

Comment:
Adding in the 'comments' flag here means blark will allow comments to be associated with the line - otherwise they'll get associated with the next rule that has it set:
```suggestion
@_rule_handler("function_call_statement_list", comments=True)
```